automation1App/src/Automation1MotorAxis.cpp

```diff
@@ -423,6 +424,12 @@ void Automation1MotorAxis::logError(const char* driverMessage)
     char errorMessage[1024];
     Automation1_GetLastErrorMessage(errorMessage, 1024);
 
+    // Check error code for "Axis not enabled" error (57000) and set "problem" status bit in that case.
+    if(errorCode == 57000)
```

Comment:
Without knowing much about EPICS, from Aerotech's perspective it seems like we should be setting the problem bit for any error, not just "axis is not enabled". For example, if the axis is currently in fault and you try to move an axis the same behavior will occur (the axis doesn't move with no error).

If it makes more sense to have EPICS check only certain errors, instead of catching all errors, then some other errors to check might be 5000 ("axis is in fault")and 5001 ("axis faulted during move"),

Reply:
I will test this and add more comments.

Setting the problem bit in EPICS usually causes the poller to stop polling, if I remember correctly, which can make it hard to recover if the driver doesn't also clear the problem bit when the problem is resolved.

The ensemble driver was modified so that it acknowedged faults when the CNEN field is set to Enable, which has historically allowed users to recover from controller problems, but that hasn't worked reliably lately.

Reply:
I remembered incorrectly.  The driver doesn't stop polling when the problem bit is set.

Reply:
I think it is ok for EPICS to set the problem bit for any error.  If there are faults that a controller should ignore, I assume the best place to configure that is in the Automation1 controller's fault setup, like we've done for older Aerotech controllers.